lib/ibandit/iban.rb

```diff
@@ -294,7 +294,6 @@ def passes_country_specific_checks?
       case country_code
       when "DE" then supports_iban_determination?
       when "SE" then valid_swedish_details?
-      when "AU" then valid_australian_details?
```

Comment:
Why has this been deleted?

Reply:
Well.. This is what happens when you forgot to self-review your PRs...